src/main/java/com/team4/testingsystem/dto/TestsRequest.java

```diff
@@ -0,0 +1,84 @@
+package com.team4.testingsystem.dto;
+
+public class TestsRequest {
+
+    private String createdAt;
+    private String updatedAt;
+    private String startedAt;
+    private String finishedAt;
+    private String status;
+    private int evaluation;
+
+    private TestsRequest(){
+
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public String getUpdatedAt() {
+        return updatedAt;
+    }
+
+    public String getStartedAt() {
+        return startedAt;
+    }
+
+    public String getFinishedAt() {
+        return finishedAt;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public int getEvaluation() {
+        return evaluation;
+    }
+
+    public static Builder builder() {
+        return new TestsRequest().new Builder();
+    }
+
+    public class Builder {
+
+        private Builder() {
+        }
+
+        public Builder setCreatedAt(String createdAt){
+            TestsRequest.this.createdAt = createdAt;
```

Comment:
Pretty strange constructions. Can it be simplified?

Reply:
I forgot to delete this file :)